Venmo/src/main/java/com/braintreepayments/api/VenmoApi.java

```diff
@@ -27,6 +27,7 @@ void createPaymentContext(@NonNull final VenmoRequest request, String venmoProfi
             input.put("merchantProfileId", venmoProfileId);
             input.put("customerClient", "MOBILE_APP");
             input.put("intent", "CONTINUE");
+            input.put("isFinalAmount", request.getIsFinalAmountAsString());
```

Comment:
Does this need to be a string? I think you an just put a boolean into the JSONObject

Reply:
The underlying API needs a string value, same as these two above that are bools on our end but strings for the API:
```java
paysheetDetails.put("collectCustomerShippingAddress", request.getCollectCustomerShippingAddressAsString());
paysheetDetails.put("collectCustomerBillingAddress", request.getCollectCustomerBillingAddressAsString());
```